mne/channels/layout.py

```diff
@@ -763,7 +766,7 @@ def plot(self, scale_factor=1.5, show_names=False):
                             show_names=show_names)
 
 
-def read_montage(kind, ch_names=None, path=None, scale=True):
+def read_montage(kind, ch_names=None, path=None, scale=None, transform=False):
```

Comment:
I would use `unit` as @agramfort suggested, maybe to be completely unambiguous `input_unit`. Also I winder whether it would make more sense to put it after `transform` and only apply it if `transform==True`.